docs/FAQ/gui-faq-en.md

```diff
@@ -4,163 +4,178 @@ Zonemaster
-(ns.example.org) *before* you move your domain.
-When the results of the test are colour coded in green one can be fairly certain
-that the domain's new location is working well. However there
-might still be other problems in the zone data itself that this test is unaware of.
+An undelegated domain test is a test performed on a domain name that may, or may not,
+be fully published in the DNS.
+This can be quite useful if one is going to migrate one's domain from one registrar to another,
+e.g., migrate zone 'example.com' from the name server 'ns.example.com' to the name server 'ns.example.org'.
+In this scenario one could perform an undelegated domain test providing the zone ('example.com')
+and the name server you are migrating to ('ns.example.org') *before* you migrate your domain.
+When the results of the test are color-coded "green" one can be fairly certain that the
+domain's new location is working well.
+However there might still be other problems in the zone data itself that this test is unaware of.
 
 <a name="q13"></a>
-#### 13. How can I test a "reverse" zone with Zonemaster?
-To check a reverse zone with Zonemaster one need to first know what the
-reverse zone is. If you want to check the reverse zone, you have to enter
-it in the format that it has in DNS, e.g.:
-
-  - 3.2.1.in-addr.arpa
-  - 6.0.1.0.0.2.ip6.arpa
-
-
+#### 13. Can I test the DS records before they are published?
+Yes.
+Use the "Options" button and there add the Delegation Signer (DS) records to be tested.
+Zonemaster will then use those in the same way as if they were already added in the parent zone.
+
+<a name="q14"></a>
+#### 14. How can I test a "reverse" zone with Zonemaster?
+To check a reverse zone with Zonemaster, one first needs to know what the
+reverse zone is, and enter it in the format it has in the DNS.
+A reserve zone is obtained by reversing an IP address and adding a suffix.
+IPv4 addresses use the suffix "in-addr.arpa" while IPv6 addresses
+use "ip6.arpa".
+
+Examples:
+  - For IPv4 prefix '198.51.100.0/24': 100.51.198.in-addr.arpa
+  - For IPv6 prefix '2001:db8::/32': 8.b.d.0.1.0.0.2.ip6.arpa
+
+[AFNIC]:                                 https://www.afnic.fr/en/
+[Defined Test Cases]:                    https://github.com/zonemaster/zonemaster/tree/master/docs/specifications/tests#list-of-defined-test-cases
+[Implemented Test Cases]:                https://github.com/zonemaster/zonemaster/blob/master/docs/specifications/tests/ImplementedTestCases.md
+[Question 12]:                           #q12
+[RFCs]:                                  https://www.ietf.org/standards/rfcs/
+[Severity Level Definitions]:            https://github.com/zonemaster/zonemaster/blob/master/docs/specifications/tests/SeverityLevelDefinitions.md
+[Swedish Internet Foundation]:           https://internetstiftelsen.se/en/
+[Test Requirements]:                     https://github.com/zonemaster/zonemaster/blob/master/docs/requirements/TestRequirements.md
```

Comment:
On next line, add:
```
[Using The CLI]: https://github.com/zonemaster/zonemaster-cli/blob/master/USING.md
```

Suggested for FAQ 11.

Reply:
Updated